app/models/card.rb

```diff
@@ -15,26 +19,10 @@ class Card < ActiveRecord::Base
 
   mount_uploader :image, CardImageUploader
 
-  scope :pending, -> { where('review_date <= ?', Time.now).order('RANDOM()') }
+  scope :pending, -> { where('review_date <= ?',
+                      Time.zone.now).order('RANDOM()') }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 23, 56 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.